src/data/reducers/index.js

```diff
@@ -0,0 +1,9 @@
+import { combineReducers } from 'redux';
+
+import connectionStatus from './connectionStatus';
+
+const rootReducer = combineReducers({
```

Comment:
`combineReducers` multiplexes together all the reducers in the application (although currently we just have one) so they can be easily passed into the store.